### PR TITLE
Provisions 02/06

### PR DESCRIPTION
**Pull request by pyrello**
# How to test

Review PR.

### CODE REVIEW

**Comment by pyrello** [comment]
Waiting for hostmaster approval and discussion about 2 other sites.

**Comment by pyrello** [comment]
Moving ahead with approved domains.

**Review by e-marie-w — APPROVED** [review]

**Pull request merged**
